dod/src/main/java/com/revolut/rxdata/dod/DataObservableDelegate.kt

```diff
@@ -205,13 +206,10 @@ class DataObservableDelegate<Params : Any, Domain : Any> constructor(
         subject(params).onNext(Data(content = domain))
     }
 
+    @Suppress("CheckResult")
     private fun fetchFromNetwork(cachedData: Domain?, params: Params) {
-        val pendingNetworkReload = sharedNetworkRequest.getOrLoad(params)
-            .toObservable()
-            .timeout(DodGlobal.networkTimeoutSeconds, TimeUnit.SECONDS, Schedulers.io())
-            .subscribe({
-                //all done in sharedRequest
-            }, { error ->
+        val observer = object : DisposableObserver<Domain>() {
```

Comment:
it doesn't seem to be a good idea to use this abstract class directly, it's lacking some stuff in it's implementation that might be important, please try and use LambdaObserver

Reply:
That's fair, thanks for the comment, done